<ui version="4.0" >
 <class>OptionsWidgetClass</class>
 <widget class="QWidget" name="OptionsWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptionsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbCommonOptions" >
     <property name="title" >
      <string>Common notification options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="chbEnableRosterIcons" >
        <property name="text" >
         <string>Enable roster icons</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnablePopupWindows" >
        <property name="text" >
         <string>Enable popup windows</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnableTrayIcons" >
        <property name="text" >
         <string>Enable tray icons</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnableTrayActions" >
        <property name="text" >
         <string>Enable tray actions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnableSounds" >
        <property name="text" >
         <string>Enable sounds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
